app/main/helpers/suppliers.py

```diff
@@ -31,14 +31,12 @@ def get_country_name_from_country_code(country_code):
 
 def supplier_company_details_are_complete(supplier_data):
     supplier_required_fields = ['dunsNumber', 'name', 'registeredName', 'registrationCountry', 'organisationSize',
```

Comment:
Not really related to this, but we have a small handful of suppliers in the db that don't have DUNS numbers. Not sure how. And on the supplier details page, we don't show the DUNS number row if the supplier doesn't have one, so they'd never be able to add one, meaning they could never complete their details.
Do you know why we have suppliers without DUNS? And why the supplier details page conditionally shows the DUNS row if we expect all suppliers to have one?

Reply:
Why...?  "Historical reasons", in that I think they are accounts that got ported from the old system that for some unknown reason were missing DUNS.  But good spot that we need to not break for these.

You're right about conditionally hiding it - should always just be there.  Will fix... somehow.